ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```
@@ -53,6 +53,81 @@ tests:
   container:
     clone: true
     from: golangci-lint
+- as: check-commits-count
+  commands: ci/prow/check-commits-count
+  container:
+    clone: true
+    from: src
+  optional: true
+- as: check-api-changes
+  commands: ci/prow/check-api-changes
+  container:
+    clone: true
+    from: src
+  optional: true
+- as: e2e-incluster-build-kernel-module
+  cluster_claim:
+    architecture: amd64
+    as: custom
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 40m0s
+    version: "4.11"
+  steps:
+    test:
+    - as: incluster-build
+      cli: latest
+      commands: |
+        KUBECONFIG=${KUBECONFIG} && GOFLAGS= ci/prow/e2e-incluster-build
```

```suggestion
        unset GOFLAGS
        oc project default
        ci/prow/e2e-incluster-build
```

I think that the CI infra still remember the namespace set in the hub cluster, but in the ephemeral cluster it just doesn't exist.

---

Nice!